CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+## Contributing to the JSON Schema Website
+
+This document outlines how you can get involved and help improve the JSON Schema website. We appreciate all contributions!
+
+## Table of Contents
+
+- [Code of Conduct 📜](#code-of-conduct-📜)
+- [How to Contribute 🚀](#how-to-contribute-🚀)
+    - [Reporting Bugs 🐞](#reporting-bugs-🐞)
+    - [Suggesting Features 💡](#suggesting-features-💡)
+    - [Improving the Website 🌐](#improving-the-website-🌐)
+- [Pull Requests 📬](#pull-requests-📬)
+- [License 📄](#license-📄)
+- [Feedback 💬](#feedback-💬)
+
+##  Code of Conduct 📜
+
+Before contributing, please familiarize yourself with our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+
+
+##  How to Contribute 🚀
+
+There are several ways you can contribute to the JSON Schema website:
+
+* [**Reporting Bugs** 🐞](#reporting-bugs-🐞)
+* [**Suggesting Features** 💡](#suggesting-features-💡)
+* [**Improving the Website** 🌐](#improving-the-website-🌐)
+
+### Reporting Bugs 🐞
+
+Encountered a bug on the website? Please use the issue templates to report it. The templates provide guidance on the information needed to help us resolve the issue.
+
+### Suggesting Features 💡 
+
+Do you have an idea for a new feature that could improve the website? Open an issue to share your suggestion.  Refer to the "Improving the Website" section for details on creating issues.
+
+## Improving the Website 🌐
+
+We welcome your help in making the JSON Schema Landscape website even better! Here's how you can contribute:
+
+#### **Improve the Content:**
+
+* See a mistake in the documentation? Have a suggestion to make it clearer? You can directly edit the Markdown files (used for the readme and contributing guide) in a pull request.
+* **Note:** Most website content uses YAML files, a data format, instead of Markdown (used for writing).
+
+**Limited Website Functionality Changes:**
+
+The website is built with a tool called CNCF Landscape, designed for managing this type of information. While large code edits aren't possible, some customization is achievable through configuration and themes.
+
+* Have an idea for a new feature that works within CNCF Landscape's limitations? Open an issue to discuss your proposal. We use a project board to track website contribution tasks: link to project board: [https://github.com/orgs/json-schema-org/projects/19](https://github.com/orgs/json-schema-org/projects/19).
+
+**Getting Started with Content or Design:**
+
+These steps are for improvements that don't involve code changes:
+
```

Comment:
Pls. fix the numbering here.

Reply:
Hi, @DhairyaMajmudar! 
In Markdown formatting, lists automatically adjust their numbering. You can check this by using the markdown preview mode, which should show the correct numbering.